client/mirage/factories/project.js

```diff
@@ -35,6 +60,7 @@ export default Factory.extend({
 
       return [
         {
+          id: `applicant-${i}`,
```

Comment:
@allthesignals  Is this the actual id format that the API will assign package models?

Reply:
Nope, this is from an old commit. I made changes along the way to make tests work. Think of it as an incremental step.